docs/tutorials/python/tutorial_scripts/evaluation.py

```diff
@@ -0,0 +1,71 @@
+"""
+Here is where you'll find the code for the Evaluation tutorial.
+"""
+
+from synapseclient import Synapse
+from synapseclient.models import Evaluation, Project
+
+syn = Synapse()
+syn.login()
+
+# REQUIRED: Set this to the Synapse user ID or team ID you want to grant permissions to
+# Do NOT leave this as None - the script will not work properly
+PRINCIPAL_ID = None  # Replace with actual user/team ID
+
+# Retrieve the Project where your Evaluation will be stored
+project = Project(name="My uniquely named project about Alzheimer's Disease").get()
+project_id = project.id
+
+print(f"Working within Project: {project_id}")
+
+# Create a new Evaluation object
+evaluation = Evaluation(
+    name="My Challenge Evaluation - Round 1",
+    description="Evaluation for my data challenge",
+    content_source=project_id,
+    submission_instructions_message="Submit CSV files only",
+    submission_receipt_message="Thank you for your submission!",
+)
+
+# Create the Evaluation on Synapse
+evaluation.store()
+
+print("Evaluation has been created with the following name and description:")
+print(evaluation.name)
+print(evaluation.description)
+
+# Update the Evaluation object's name and description
+evaluation.name = "My Challenge Evaluation - Round 2"
+evaluation.description = "Updated description for my evaluation"
```

Comment:
This is good to show that one can update these values to something new - Logically though, is that how a new "Round" of challenges is run? I would assume that a new evaluation would be created for an additional round.

Reply:
That makes sense: Updated! https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1247/commits/48e6109eaadb8df2d6b26a7b86e10e02cc01fbe6